server/__init__.py

```diff
@@ -37,7 +37,7 @@ def add_api_endpoints(app):
         methods=["POST"],
     )
     app.add_url_rule(
-        "/git/",
+        "/git//<string:platform_name>/" "<string:organisation_name>/<int:project_id>/",
```

Comment:
Is the double slashs correct? `/git//<string:platform_name>/`

Reply:
It wasn't intentional, I updated to use just one slash. Thanks for noticing it!